<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Триатлон</string>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Время</string>
    </property>
   </widget>
   <widget class="QLabel" name="stone">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Камушек</string>
    </property>
   </widget>
   <widget class="QLabel" name="cactoo">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Кактус</string>
    </property>
   </widget>
   <widget class="QLabel" name="donkey">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Осел</string>
    </property>
   </widget>
   <widget class="QLabel" name="run">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Бег</string>
    </property>
   </widget>
   <widget class="QLabel" name="swim">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Плавание</string>
    </property>
   </widget>
   <widget class="QLabel" name="bike">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Велосипед</string>
    </property>
   </widget>
   <widget class="QLabel" name="AllTime">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>общее время</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stone_run">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="stone_swim">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="stone_bike">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="stone_allTime">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cactoo_run">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cactoo_swim">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cactoo_bike">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cactoo_allTime">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>440</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="donkey_run">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="donkey_swim">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="donkey_bike">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="donkey_allTime">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>440</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="score">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Счет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="champion">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Чемпион</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="champions">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>520</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
